datalad/support/tests/test_gitrepo.py

```diff
@@ -414,10 +414,10 @@ def test_GitRepo_fetch(test_path, orig_path, clone_path):
 
     fetched = clone.fetch(remote='origin')
     # test FetchInfo list returned by fetch
-    eq_([u'origin/master', u'origin/new_branch'],
+    eq_([u'origin/' + clone.get_active_branch(), u'origin/new_branch'],
         [commit.name for commit in fetched])
 
-    ok_clean_git(clone.path, annex=False)
+    ok_clean_git(clone.path)
```

Comment:
so, before we were explicitly testing that fresh clone is not an annex (since no annex commands were ran).  That behavior was tested explicitly with `annex=False`.  Current new default is `annex=None` which allows either if I have got it right. So it relaxes this (and other similar) test, what for?